From 69bc30bbfa41c4730ca786076fa425a18c54b155 Mon Sep 17 00:00:00 2001
From: compulab <compulab@CompuLab.com>
Date: Tue, 12 Aug 2025 18:15:57 +0000
Subject: [PATCH 58/58] ucm-imx8m-mini:rev1.4:dtb: Free uart4

Delete Bluetooth node, so as free uart4. This makes NXP Bluetooth interface able to be brought by means of hcitools, as per ch. 6.2 of UM11483  https://www.nxp.com/docs/en/user-manual/UM11483.pdf

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/wifi-iw612.dtsi | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/wifi-iw612.dtsi b/arch/arm64/boot/dts/compulab/wifi-iw612.dtsi
index 195968a435cb..925f2600634e 100644
--- a/arch/arm64/boot/dts/compulab/wifi-iw612.dtsi
+++ b/arch/arm64/boot/dts/compulab/wifi-iw612.dtsi
@@ -1,12 +1,3 @@
-&uart4 { /* bluetooth */
-	bluetooth_iw612: bluetooth {
-		compatible = "nxp,88w8987-bt";
-		//fw-init-baudrate = <3000000>; //Optional. Default is considered 115200 if this parameter not defined.
-		max-speed = <3000000>;
-		status = "okay";
-	};
-};
-
 &usdhc1 {
 	wifi_iw612: wifi_wake_host {
 		compatible = "nxp,wifi-wake-host";
-- 
2.17.1

